docs/README.md

```diff
@@ -35,6 +35,17 @@ language, grammar, and tone style:
 * If using an abbreviation, spell it out the first time it is used in the document unless it is commonly known. (example: TCP/IP)
 * When referring to a Kubernetes Group (SIG, WG, or UG) do not use the hyphenated form unless it is for a specific purpose such as a file-name or URI.
 
+## reStructured Text Styling Guide
```

Comment:
This section should appear after line 49 i.e. the cheatsheet from the Kubernetes community style guide is more relevant to the language usage, than the formatting.